modules/backend/models/ExportModel.php

```diff
@@ -135,13 +138,10 @@ protected function processExportData($columns, $results, $options)
         }
 
         /*
-         * Save for download
+         * Save in cache for download
          */
         $csvName = uniqid('oc');
-        $csvPath = temp_path().'/'.$csvName;
-        $output = $csv->__toString();
-
-        File::put($csvPath, $output);
+        Cache::put('export::'.$csvName, $csv->__toString(), 5);
```

Comment:
@Samuell1 Please add space between concatenation parts.

```suggestion
        Cache::put('export::' . $csvName, $csv->__toString(), 5);
```